ArcGISRuntimeSDKQt_CppSamples/Layers/FeatureLayerShapefile/FeatureLayerShapefile.json

```diff
@@ -0,0 +1,8 @@
+{
+    "source" :  "FeatureLayerShapefile.qml",
+    "otherSourceFiles" :  [
+        "FeatureLayerShapefile.h",
+        "FeatureLayerShapefile.cpp"
+    ],
+    "name" : "Feature layer (shapefile)"
```

Comment:
this needs a dataItems section so that the viewer knows to download data with it - e.g. https://github.com/Esri/arcgis-runtime-samples-qt/blob/v.next/ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/SymbolizeShapefile/SymbolizeShapefile.json#L8-L13